src/apns_connection.erl

```diff
@@ -143,9 +143,9 @@ handle_info(_Info, State) ->
 
 -spec terminate( Reason :: (normal | shutdown | {shutdown, term()} | term())
                , State  :: map()
-               ) -> term().
+               ) -> ok.
 terminate(_Reason, #{gun_connection := GunConn}) ->
-  ok = gun:shutdown(GunConn),
+  gun:shutdown(GunConn),
   ok.
```

Comment:
My goal was to also remove this line :P - If you're going to leave it, then the pattern matching above can stay :)